docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -242,22 +242,56 @@ kubectl logs -f alluxio-worker-<id> -c alluxio-job-worker
 ### Short-circuit Access
 
 Short-circuit access enables clients to perform read and write operations directly against the
-worker bypassing the networking interface. As part of the Alluxio worker pod creation, a directory
-is created on the host at `/tmp/domain` for the shared domain socket.
+worker bypassing the networking interface.
+For performance-critical applications it is recommended to enable short-circuit operations
+against Alluxio because it can increase a client's read and write throughput when co-located with
+an Alluxio worker.
+
+#### Properties to Enable Short-Circuit Operations
+
+This feature is enabled by default, however requires extra configuration to work properly in
+Kubernetes environments.
+To disable short-circuit operations, set the property `alluxio.user.short.circuit.enabled=false`.
+
+##### Hostname Introspection
 
-To disable this feature, set the property `alluxio.user.short.circuit.enabled=false`. By default,
-short-circuit operations between the Alluxio client and worker are enabled if the client hostname
-matches the worker hostname. This may not be true if the client is running as part of a container
-with virtual networking. In such a scenario, set the following property to use filesystem inspection
-to enable short-circuit and make sure the client container mounts the directory specified as the
-domain socket address. Short-circuit writes are then enabled if the worker UUID is located on the
+Short-circuit operations between the Alluxio client and enabled by default worker are enabled if the client
+hostname matches the worker hostname.
+This may not be true if the client is running as part of a container with virtual networking.
+In such a scenario, set the following property to use filesystem inspection to enable short-circuit
+and **make sure the client container mounts the directory specified as the domain socket path**.
+Short-circuit writes are then enabled if the worker UUID is located on the
 client filesystem.
 
+> Note: This property should be set on all workers
+
 ```properties
 alluxio.worker.data.server.domain.socket.as.uuid=true
+```
+
+##### Domain Socket Path
+
+The domain socket is a volume which should be mounted on:
+
+- All Alluxio workers
+- All application containers which intend to read/write through Alluxio
+
+The exact path of the domain socket is defined in the Kubernetes templates at
+`${ALLUXIO_HOME}/integration/kubernetes/alluxio-worker.yaml.template` and
+`${ALLUXIO_HOME}/integration/kubernetes/alluxio-fuse.yaml.template`.
+By default this value is `/opt/domain`.
```

Comment:
same here, my bad. `/tmp/alluxio-domain` should be replaced by `/opt/domain` even in master